src/tex/appendix.tex

```diff
@@ -447,6 +447,18 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone model consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.47 on \ChemBench corpus, which is only 18\% less accurate than the larger \LlamaThreeOneEightBInstruct model.
+One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo). Newer versions consistently outperform their predecessors, indicating improvements in scientific understanding of these model families.
+Scores of knowledge-intensive are generally lower than those of Calculation and Reasoning. Reasoning tends to be the strongest skill among the three.
+
+Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. 
```

Comment:
```suggestion
Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. 
```

perhaps link to this facebook paper that stuff is learned in pretraining

Reply:
Perhaps says that just training on ChemRxiv or stuff like this will not help - link to the performance of PaperQA
And that much of the knowledge chemists care about is also very specific (e.g. about certain reactions or molecules for which chemists also go to specialized databases)

Reply:
Perhaps say that finding out training tasks that instill a clearer molecular representation would be a way forward